pyne/alara.py

```diff
@@ -517,3 +519,149 @@ def irradiation_blocks(material_lib, element_lib, data_library, cooling,
+                   complex(46.933274488831293047, 45.643649768827760791), 
+                   complex(-27.875161940145646468, -102.14733999056451434), 
+                   complex(4.8071120988325088907, -1.3209793837428723881)])
+    
+    alpha_0 = np.array([1.8321743782540412751*10**(-14)])
+
+    s = 7
+    A = A*t
+    n = 0*n_0
+
+    for j in range(7):
+        n = n + np.dot(alpha[j]*n_0, np.linalg.inv(A - theta[j] * np.identity(np.shape(A)[0])))
+
+    n = 2*n.real
+    n = n + alpha_0*n_0
+    
+    return n
+
+def _rat_apprx_16(A, t, n_0):
+    """ CRAM of order 16
+
+    Parameters
+    ---------
+
+    A : numpy array
+        Burnup matrix
+    t : float
+        Time step
+    n_0: numpy array
+        Inital composition vector
+    """
+    theta = np.array([ complex(-10.843917078696988026, 19.277446167181652284), 
+                   complex(-5.2649713434426468895, 16.220221473167927305), 
+                   complex(5.9481522689511774808, 3.5874573620183222829), 
+                   complex(3.5091036084149180974, 8.4361989858843750826), 
+                   complex(6.4161776990994341923, 1.1941223933701386874), 
+                   complex(1.4193758971856659786, 10.925363484496722585), 
+                   complex(4.9931747377179963991, 5.9968817136039422260), 
+                   complex(-1.4139284624888862114, 13.497725698892745389)])
+    
+    alpha = np.array([ complex(-.0000005090152186522491565, -.00002422001765285228797), 
+                   complex(.00021151742182466030907, .0043892969647380673918), 
+                   complex(113.39775178483930527, 101.9472170421585645), 
+                   complex(15.059585270023467528, -5.7514052776421819979), 
+                   complex(-64.500878025539646595, -224.59440762652096056), 
+                   complex(-1.4793007113557999718, 1.7686588323782937906), 
+                   complex(-62.518392463207918892, -11.19039109428322848), 
+                   complex(.041023136835410021273, -.15743466173455468191)])
+    
+    alpha_0 = np.array([2.1248537104952237488*10**(-16)])
+    
+
+    s = 8
+    A = A*t
+    n = 0*n_0
+
+    for j in range(8):
+        n = n + np.dot(alpha[j]*n_0, np.linalg.inv(A - theta[j] * np.identity(np.shape(A)[0])))
+
+    n = 2*n.real
+    n = n + alpha_0*n_0
+    return n
+
+def CRAM(N, t, n_0, order):
+    """ This function returns matrix exponential solution n using CRAM14 or CRAM16
+
+    Parameters
+    ----------
+
+    N : list or array
+	Array of nuclides under consideration
+    t : float
+	Time step
+    n_0 : list or array
+	Nuclide concentration vector
+    order : int
+	Order of method. Only 14 and 16 are supported.
+    """
+
+    n_0 = np.array(n_0)
+    A = build_matrix(N)
+    
+    if order == 14:
+        return rat_apprx_14(A, t, n_0)
+    
+    if order == 16:
+        return rat_apprx_16(A, t, n_0)
+    
+    else:
+        return 'Rational approximation of degree %d is not supported.' % order
```

Comment:
This should raise a ValueError with this message.  Also don't use `%` for string formatting.  I would rewrite this as:

```python
msg = 'Rational approximation of degree {0} is not supported.'.format(order)
raise ValueError(msg)
```